### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by vjandrea**
Modified the controller example to be ready-to-use

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,15 +82,21 @@ Just inject the service by type-hinting an argument with the `Jupi\PrestaShopWeb
...
 
-class ProductController
+class ProductController extends AbstractController
 {
     /**
      * @Route("/products")
+     *
+     * @param PrestaShopWebservice $psWebservice
+     *
+     * @return Response
```
Comment (Jupi007) [inline README.md]: I don't think this is needed, we already have param and return types in native PHP.
And this add too much lines imo, I would prefer to keep the example as small as possible.

**Review by Jupi007 — CHANGES_REQUESTED** [review]
Thanks a lot for the PR @vjandrea ❤️

File: README.md (hunk 1 above)
Comment (Jupi007) [reply README.md]: Also, if you want to make the change. I was thinking to convert the multiline phpdoc `@Route` in a single line PHP annotation like this: `#[Route('/products', name: 'products')]`

Comment (vjandrea) [reply README.md]: Ok I agree!

**Review by Jupi007 — APPROVED** [review]
Perfect 🚀

**Pull request merged**